ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,249 @@
+﻿[[!meta title="Getting Involved"]]
+
+## How can I get involved in the OCF?
+
+Glad you asked! Students from all backgrounds are welcome to join staff. We
+don’t interview or have a selective process for joining—you simply have to
+express interest and show up to our weekly meetings. We don't impose a specific
+time commitment; some volunteers work many hours per week, others contribute
+occasionally. If you want to learn more and be an active participant, there are
+multiple ways to get started:
+
+### Stay up-to-date
+
+We use a mailing list ([ocf.io/announce][announce]) to announce weekly meetings
+and recap past meetings. If you’re joining in the middle of the semester or
+missed a meeting, you can also read the mailing list archives to see what we
```

Comment:
It'd be good to link to the archives here.

Reply:
ocf.io/announce is the archives